packages/controller-utils/src/constants.ts

```diff
@@ -98,9 +98,7 @@ export const BUILT_IN_NETWORKS = {
 
 // APIs
 export const OPENSEA_PROXY_URL =
-  'https://proxy.metafi.codefi.network/opensea/v1/api/v1';
-export const OPENSEA_API_URL = 'https://api.opensea.io/api/v1';
-export const OPENSEA_TEST_API_URL = 'https://testnets-api.opensea.io/api/v1';
```

Comment:
Can you note the removal of `OPENSEA_TEST_API_URL` as a breaking change in the PR description as well?

Reply:
yes.  as far as i can tell, the test URLs weren't used anywhere else in core, extension, or mobile.  everyone goes through proxy.